io/plugins/eu.esdihumboldt.hale.io.jdbc/src/eu/esdihumboldt/hale/io/jdbc/connection/NoCloseConnection.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2024 wetransform GmbH
+ *
+ * All rights reserved. This program and the accompanying materials are made
+ * available under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution. If not, see <http://www.gnu.org/licenses/>.
+ */
+package eu.esdihumboldt.hale.io.jdbc.connection;
+
+import java.sql.Connection;
+
+public class NoCloseConnection extends ConnectionDecorator {
```

Comment:
A short JavaDoc section might be helpful here as well

```suggestion
/**
 * Decorator for {@link Connection}s that ignores calls to {@link #close()}.
 */
public class NoCloseConnection extends ConnectionDecorator {
```